### PR TITLE
NOX-340 Localization endpoints have been added to doc generator

### PR DESCRIPTION
**Pull request by msadikkoseaupa**

### CODE REVIEW

File: src/Nox.Docs/Templates/EntityEndpoints.template.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ This document provides information about the various endpoints available in our
 ### Get {{entity.PluralName}}
 - **GET** `/api/{{entity.PluralName}}`
   - Description: Retrieve information about {{entity.PluralName}}.
+{{~ if entity.IsLocalized ~}}
```
Comment (gjorgji-grgovski-authoritypartners) [inline src/Nox.Docs/Templates/EntityEndpoints.template.md]: Can you add those endpoints at the end within a section "Localized Endpoints" and with only one If statement which would wrap all endpoints?

**Review by gjorgji-grgovski-authoritypartners — CHANGES_REQUESTED** [review]

File: src/Nox.Docs/Templates/EntityEndpoints.template.md (hunk 1 above)
Comment (msadikkoseaupa) [reply src/Nox.Docs/Templates/EntityEndpoints.template.md]: Desired changes have been applied

File: tests/Nox.Docs.Tests/Nox.Docs.Tests/Extensions/NoxSolutionEntityEndpointsMarkdownExtensionsTests.cs [hunk 2]
```diff
@@ -18,6 +18,12 @@ public void Solution_Creates_Valid_EntityEndpoints_Markdown()
         // Act
         var actual = noxSolution.ToMarkdownEntityEndpoints()
             .OrderBy(m => m.Name);
+        
+        
+        foreach (var item in actual)
```
Comment (rochar) [inline tests/Nox.Docs.Tests/Nox.Docs.Tests/Extensions/NoxSolutionEntityEndpointsMarkdownExtensionsTests.cs]: you forgot to remove this?

**Review by rochar — CHANGES_REQUESTED** [review]
looks good

File: tests/Nox.Docs.Tests/Nox.Docs.Tests/Extensions/NoxSolutionEntityEndpointsMarkdownExtensionsTests.cs (hunk 2 above)
Comment (msadikkoseaupa) [reply tests/Nox.Docs.Tests/Nox.Docs.Tests/Extensions/NoxSolutionEntityEndpointsMarkdownExtensionsTests.cs]: Oh yes, removed it

**Review by rochar — APPROVED** [review]

**Review by gjorgji-grgovski-authoritypartners — APPROVED** [review]

**Pull request merged**